src/dscanner/analysis/unused_label.d

```diff
@@ -5,128 +5,119 @@
 module dscanner.analysis.unused_label;
 
 import dscanner.analysis.base;
-import dscanner.analysis.helpers;
-import dparse.ast;
-import dparse.lexer;
-import dsymbol.scope_ : Scope;
-import std.algorithm.iteration : each;
+import dmd.tokens;
 
 /**
  * Checks for labels that are never used.
  */
-final class UnusedLabelCheck : BaseAnalyzer
+extern(C++) class UnusedLabelCheck(AST) : BaseAnalyzerDmd
 {
-	alias visit = BaseAnalyzer.visit;
-
+	alias visit = BaseAnalyzerDmd.visit;
 	mixin AnalyzerInfo!"unused_label_check";
 
-	///
-	this(string fileName, const(Scope)* sc, bool skipTests = false)
+	extern(D) this(string fileName, bool skipTests = false)
 	{
-		super(fileName, sc, skipTests);
+		super(fileName, skipTests);
 	}
 
-	override void visit(const Module mod)
+	override void visit(AST.Module m)
```

Comment:
Is this visiting method really necessary? Labels cannot be put at module scope so performing a check there doesn't really make sense.